flake.nix

```diff
@@ -44,15 +41,6 @@
             name = "chez-monadic";
             buildInputs = [pkgs.chez];
           };
-          chez-lift = {
-            name = "chez-lift";
-            buildInputs = [pkgs.chez];
-          };
-        } // pkgs.lib.optionalAttrs isMLtonSupported {
-          ml = {
-            name = "ml";
-            buildInputs = [pkgs.mlton];
-          };
```

Comment:
We could also provide nice deprecation messages, but -- as far as I know -- not a lot of people who use this also use chez-lift and/or MLton, especially since both are currently broken in different ways in this flake.